### PR TITLE
Create insertJavaScript method

### PR DESCRIPTION
**Pull request by snowball**
To fix https://github.com/SBoudrias/gruntfile-editor/issues/9, feedback appreciated!

### CODE REVIEW

**Comment by snowball** [comment]
@SBoudrias any feedback? Able to merge?

**Comment by SBoudrias** [comment]
Hey, this is in my pipeline, I'm just a bit busy for reviews/coding lately.

File: test/insertjavascript.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      assert.throws(this.insert(42), msg);
+      assert.throws(this.insert(false), msg);
+      assert.throws(this.insert(true), msg);
+      assert.throws(this.insert(0), msg);
+      assert.throws(this.insert(-1), msg);
+      assert.throws(this.insert([ 'an', 'array' ]), msg);
+    });
+  });
+
+  describe('inserts', function () {
```
Comment (SBoudrias) [inline test/insertjavascript.js]: no need for this describe block.

File: test/insertjavascript.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    this.editor = new GruntfileEditor();
+    this.insert = helper.insertJavaScript;
+    this.str = helper.str;
+  });
+
+  it('is chainable', function () {
+    assert.equal(this.editor.insertJavaScript('require(\'load-grunt-tasks\')(grunt);'), this.editor);
+  });
+
+  describe('requires code', function () {
```
Comment (SBoudrias) [inline test/insertjavascript.js]: no need for this describe block - and no need for a scoped variable. Just make it part of the `it` block.

**Comment by SBoudrias** [comment]
I think it's be better if we renamed this method `prependJavaScript()` (as we could add a method to append code too).

Also, can we update `insertVariable` to also use this method?

File: test/insertjavascript.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+var assert = require('assert');
+var GruntfileEditor = require('..');
+var helper = require('./helpers');
+
+describe('#insertJavaScript()', function () {
+
+  beforeEach(function () {
+    this.editor = new GruntfileEditor();
+    this.insert = helper.insertJavaScript;
```
Comment (SBoudrias) [inline test/insertjavascript.js]: I don't think this value is defined.

Comment (snowball) [inline test/insertjavascript.js]: It should be, otherwise the is chainable test should fail

**Comment by snowball** [comment]
Thanks for your feedback, have made changes requested, let me know if it looks ok :)

**Comment by SBoudrias** [comment]
Alright, that LGTM. Can you squash your commits in a single one?

**Comment by snowball** [comment]
Done

**Comment by SBoudrias** [comment]
Nice! Thanks a lot!

**Pull request merged**